### PR TITLE
Add repository health badges

### PR DESCRIPTION
**Pull request by peternied**
### Description
Adding more detailed badges generated with
https://github.com/peternied/opensearch-build-badger

### Issues Resolved
- Related https://github.com/opensearch-project/opensearch-build/issues/2629

### What is needed to leave draft state
- [X] Basic agreement that we can move forward with the visual style, haven't see any pushback

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peternied** [comment]
For better review see https://github.com/peternied/security/tree/manually-badged#readme

**Review by DarshitChanpura — APPROVED** [review]

**Review by cwperks — DISMISSED** [review]
The change looks good to me. Can you re-base this with the latest build fixing changes from main?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
...
+[![](https://img.shields.io/codecov/c/gh/opensearch-project/security)](https://app.codecov.io/gh/opensearch-project/security) [![](https://img.shields.io/github/issues/opensearch-project/security/v2.4.0)](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3A"v2.4.0") [![](https://img.shields.io/github/issues/opensearch-project/security/v3.0.0)](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3A"v3.0.0")
```
[the remark this answers is not in the document]
Comment (peternied) [reply README.md:2]: There is an interesting question, how is the destination of these badges controlled.  This readme file is located on the default 'main' branch.  Could it show the next 2.x minor and 1.3.x patch, what happens if those versions increment?

I've been separately building a way to identify and trigger an updated 'badger' task - not yet out for PR.

**Review by cwperks — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (peternied) [reply README.md:2]: Note; will track this as part of https://github.com/opensearch-project/opensearch-build/issues/2629

**Review by DarshitChanpura — APPROVED** [review]

**Pull request merged**